### PR TITLE
add genre preprocessing

### PR DESCRIPTION
**Pull request by kaustika**
add parsing of genre from the given the meta-info file

### CODE REVIEW

File: Pipeline/Preprocessing/PreprocessingModule.py [hunk 1]
```diff
@@ -80,20 +83,24 @@ def __cut_in_pieces(path_to_audio, bin_mask, seq_len):
                 paths_to_cuts.append(path_to_cut)
 
                 mask = bin_mask[(cut - 1) * seq_len: cut * seq_len]
+                g_mask = genre_mask[(cut - 1) * seq_len: cut * seq_len]
```
Comment (OlegBEZb) [inline Pipeline/Preprocessing/PreprocessingModule.py:86]: Optional: you can make a function for not repeating yourself

File: Pipeline/Preprocessing/PreprocessingModule.py [hunk 2]
```diff
@@ -114,31 +121,55 @@ def __download_from_youtube(link, path_to_video):
...
+        }
         bin_mask = []
+        genre_mask = []
         link = None
         with open(path_to_meta) as meta:
             for line in meta:
+                line = line.strip(" \n\t").replace(' ', '')
                 if line.startswith("http"):
-                    link = line.rstrip("\n")
+                    link = line
```
Comment (OlegBEZb) [inline Pipeline/Preprocessing/PreprocessingModule.py:156]: for what?

Comment (kaustika) [reply Pipeline/Preprocessing/PreprocessingModule.py:156]: @OlegBEZb i don't quite understand what you mean. the 1st line of the file is the link to the YT video, this is where it is extracted.

Comment (OlegBEZb) [reply Pipeline/Preprocessing/PreprocessingModule.py:156]: oh, I'm sorry. I've read as `line = line`
my oversight(

**Pull request merged**